LuaRocks doesn't work with busybox wget, so prefer curl.

See https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/48613

--- a/src/luarocks/fs/tools.lua
+++ b/src/luarocks/fs/tools.lua
@@ -16,8 +16,8 @@
    local tool_options = {
       downloader = {
          desc = "downloader",
-         { var = "WGET", name = "wget" },
          { var = "CURL", name = "curl" },
+         { var = "WGET", name = "wget" },
       },
       md5checker = {
          desc = "MD5 checker",
